### DISCUSSION TITLE
Sidebar graph

### DISCUSSION THREAD

**Discussion by MrJelly17** [matt8707/ha-fusion#398] [General]
What type of sensors we can use to display a graph in the sidebar? I'm trying to set up a bitcoin sensor but it is not visible in a drop-down menu. Is it have to be any specific type of sensor?

**Comment by gimmiereddy** [comment]
I believe the sensor needs to have a "state_class: measurement".

**Reply by MrJelly17** [reply]
Thanks. I will try

**Comment by MrJelly17** [comment]
Invalid config for 'template' from integration 'sensor' at configuration.yaml, line 25: 'state_class' is an invalid option for 'sensor.template', check: sensors->btc_price_rounded->state_class

**Reply by gimmiereddy** [reply]
Take a look here [Template](https://www.home-assistant.io/integrations/template/). Judged by that error message you are using legacy template sensor. You’re probably best of rewriting it to the modern template. 

That should be fairly straightforward.
